### DISCUSSION TITLE
Is IKVM.NET.Sdk 8.8 able to create .NET 6 code?

### DISCUSSION THREAD

**Discussion by martin-honnen** [ikvmnet/ikvm#527] [Q&A]
I am wondering whether https://www.nuget.org/packages/IKVM.NET.Sdk is able to target .NET 6 as well as .NET 8 (in the .NET "Core" world) or whether it is for .NET 8 only.
Any thoughts?

**Comment by wasabii** [comment]
Either. It's what we use to build IKVM.Java itself.

**Reply by martin-honnen** [reply]
So using e.g. `<TargetFrameworks>net6.0;net8.0</TargetFrameworks>` should work in an IKVM.NET.Sdk 8.8 project?

**Reply by wasabii** [reply]
Should.

**Reply by martin-honnen** [reply]
Thanks, I tried now from the command line with dotnet build and it runs through, before I had try to let VS build and it failed. Will figure it out.
